tests/transformers/tests/models/llama/test_modeling_llama.py

```diff
@@ -17,17 +17,17 @@
 import unittest
 
 from parameterized import parameterized
-from transformers import LlamaConfig, is_torch_available
+from transformers import is_torch_available
 from transformers.testing_utils import require_torch, slow
 
 from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
+from optimum.habana.transformers.models.llama.configuration_llama import LlamaConfig
 from optimum.habana.utils import set_seed
 
 from ...generation.test_utils import GenerationTesterMixin
 from ...test_configuration_common import ConfigTester
 from ...test_modeling_common import ModelTesterMixin, ids_tensor, random_attention_mask
 
-
```

Comment:
I think this line removal is causing code style error, please put it back.

Reply:
Yes, the code formatter will enforce 2 blank lines between the imports and the rest.
Otherwise @rkumar2patel you can just run `make style` and push the changes.